msteams-platform/concepts/build-and-test/apps-package.md

```diff
@@ -56,11 +56,16 @@ Teams automatically crops your icon to display a square with rounded corners in
 
 An outline icon displays in two scenarios:
 
-* When your app is in use and “hoisted” on the app bar on the left side of Teams.
+* When your app is in use and pinned to the app bar on the left side of Teams.
 * When a user pins your app's message extension.
 
 Ensure the icon is 32x32 pixels. It should be either white with a transparent background or transparent with a white background. No other colors are allowed. The outline icon mustn't contain any additional padding around the symbol.
 
+> [!NOTE]
+> The following guidance applies only to the developer preview manifest.
+
+Although the outline icon is used when your app is pinned in Teams, when your app is pinned in Outlook or the Microsoft 365 app, the scaled down color icon is used instead. This could result in inconsistent appearances across experiences. To ensure your full color app icon looks good when pinned in Outlook and Microsoft 365 app, you can specify an [optional 32x32 color icon](~/resources/schema/manifest-schema-dev-preview.md#icons) with a transparent background.
+
 :::image type="content" source="../../assets/images/icons/design-outline-icon.png" alt-text="Teams outline icon design guidance.":::
```

Comment:
```suggestion
An outline icon for your app is displayed in Teams, Outlook and Microsoft 365 app. The following table lists the scenarios where an outline icon is used:

|Platform  |Scenario  | Required|
|---------|---------|----|
|Teams     | •	When an app is in use and pinned to the app bar. </br> •	When a user pins an app's message extension.      |✔️ |
|Outlook and Microsoft 365 (Preview)   |  When an app is pinned in Outlook or the Microsoft 365 app.        ||

Ensure that the outline icon adheres to the following best practices:

* **Teams**: The icon size must be in 32x32 pixels. It should be either white with a transparent background or transparent with a white background. No other colors are allowed. The outline icon mustn't contain any additional padding around the symbol.

* **Outlook and Microsoft 365 (Preview)**: The icon must be a [32x32 color icon](~/resources/schema/manifest-schema-dev-preview.md#icons) with a transparent background.

:::image type="content" source="../../assets/images/icons/design-outline-icon.png" alt-text="Screenshot shows the design guidance of an outline color icon.":::
```

Reply:
Can you also verify if the following is still true:

**When a user pins your app's message extension.**

thanks,
Yash

Reply:
Good catch @v-ypalikila! I don't think pinning MEs is possible anymore (new Teams). I tried various installed MEs and the only right-click context menu option is _Uninstall_ : 
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/2170770/6a15b6d1-5cb1-4ecf-aac2-06a65530a7c6). 

@mobajemu-msft let's file a Teams docs issue to remove pinning guidance on the [message extensions](https://learn.microsoft.com/en-us/microsoftteams/platform/messaging-extensions/design/messaging-extension-design) overview.

For this PR, I'd like to propose this rephrasing. Instead of:

> An outline icon displays in two scenarios:
> 
> * When your app is in use and pinned to the app bar on the left side of Teams.
> * When a user pins your app's message extension.

How about:

> An outline icon displays when your app is in use and/or pinned to the app bar on the left side of Teams.

Reply:
would it be beneficial to describe the fall back behavior in Outlook and Microsoft 365 if the optional icon is not provided? In that case we use the existing color icon which may or may not be transparent and round the corners in a manner similar to the Flyout.

Reply:
Also worth noting that Outlook also supports message extensions. Today we use the existing `color` asset but I see no reason why we wouldn't also adopt the `color32x32` asset if available in that use case.

Reply:
@krubenok does this pertain to the ME app icon from the _All _Apps_/Apps_ menu of Outlook compose window, or a different entrypoint?